components/PresidentialResults/index.js

```diff
@@ -27,15 +27,14 @@ const fetchResults = (answers) =>
   }).then((data) => data.json());
 
 export default function PresidentialResults() {
-  const { userAnswers, resetTopics } = useTopics();
+  const { userAnswers } = useTopics();
   const { data: response, error, isLoading } = useSWR(
     '/api/policies/results',
     () => fetchResults(userAnswers.filter((answer) => answer.answerId)),
   );
   const results = response?.data;
 
   const handleGoBackButton = () => {
```

Comment:
`GoBackButton` tiene una funcionalidad de to="/" que ya hace esto, en este caso `handleGoBackButton` existe porque hacía el resetTopics, borrar toda esta función, chequear los cambios de: https://github.com/openpolitica/open-wizard/pull/267/files